faster_sam/openapi.py

```diff
@@ -5,7 +5,40 @@
 
 
 def custom_openapi(app: FastAPI, openapi_schema: Dict[str, Any]) -> Callable[[], Dict[str, Any]]:
+    """
+    Custom OpenAPI generator for the FastAPI application.
```

Comment:
Sugestão adicionando mais alguns detalhes:

_Returns an OpenAPI schema generator function for FastAPI._

_The generator function uses the given input schema as the base to generate a new schema combining it with the auto-generated by FastAPI._